lib/active_model_serializers/logging.rb

```diff
@@ -0,0 +1,112 @@
+##
+# ActiveModelSerializers::Logging
+#
+#   https://github.com/rails/rails/blob/280654ef88/activejob/lib/active_job/logging.rb
+#
+module ActiveModelSerializers::Logging
```

Comment:
I was on the fence about this

It requires the ams module to be loaded first.  That's a feature, not a gotcha, in thid case

But i'm not sure how useful it us here

Copied pattern from gem;:deprecate, which is really the only reason i considered it

B mobile phone

> On Nov 9, 2015, at 6:53 AM, Mauro George <notifications@github.com> wrote:
> 
> In lib/active_model_serializers/logging.rb:
> 
> > @@ -0,0 +1,112 @@
> > +##
> > +# ActiveModelSerializers::Logging
> > +#
> > +#   https://github.com/rails/rails/blob/280654ef88/activejob/lib/active_job/logging.rb
> > +#
> > +module ActiveModelSerializers::Logging
> Any special reason to use ActiveModelSerializers::Logging over a nested module? I think the more verbose is prefered http://blog.elpassion.com/ruby-gotchas/. Makes sense?
> 
> —
> Reply to this email directly or view it on GitHub.
>